perfkitbenchmarker/linux_benchmarks/netperf_benchmark.py

```diff
@@ -377,7 +389,19 @@ def RunNetperf(vm, benchmark_name, server_ip, num_streams):
                      server_ip=server_ip,
                      length=FLAGS.netperf_test_length,
                      output_selector=OUTPUT_SELECTOR,
-                     confidence=confidence, verbosity=verbosity)
+                     confidence=confidence,
+                     verbosity=verbosity)
+
+  if benchmark_name.upper() == 'UDP_STREAM':
+    netperf_cmd += (' -R 1 -m {send_size} -M {send_size} '.format(
```

Comment:
What is `-R 1` for?  I don't see it in https://hewlettpackard.github.io/netperf/doc/netperf.html#UDP_005fSTREAM

Reply:
I see it here as response size: http://www.cs.kent.edu/~farrell/dist/ref/Netperf.html

Reply:
For posterity, it is documented in https://github.com/HewlettPackard/netperf/blob/bcb868bde7f0203bbab69609f65d4088ba7398db/Release_Notes#L212